app/templates/views/guidance/using-notify/delivery-times.html

```diff
@@ -34,9 +34,12 @@ <h2 id="letters" class="heading-medium">Letters</h2>
   <p class="govuk-body">First class letters are delivered one day after they’re dispatched. Second class letters are delivered 2 days after they’re dispatched. Royal Mail delivers from Monday to Saturday, excluding bank holidays.</p>
   <p class="govuk-body">Letters to Europe are delivered 3 to 5 days after they’re dispatched. Letters sent anywhere else in the world take 5 to 7 days to arrive.</p>
 
-  <h3 id="returned-mail" class="heading-small">Undelivered letters and returned mail</h3>
-  <p class="govuk-body">Every letter we send includes our printer’s address on the back of the envelope. It’s not possible to customise the return address.</p>
-  <p class="govuk-body">Returned mail is destroyed by our printer.</p>
-  <p class="govuk-body">Each letter has a unique ID. Our printer sends us the ID of every letter they destroy. We’ll contact you if the IDs match any of your letters. This is a partly manual process, so it can take a few weeks.</p>
+  <h3 id="returned-mail" class="heading-small">Returned letters</h3>
+  <p class="govuk-body">Every letter we send includes our print provider's address on the back of the envelope. It’s not possible to customise the return address.</p>
+  <p class="govuk-body">Undelivered letters are returned to our print provider and are destroyed.</p>
+  <p class="govuk-body">A returned letters report is available on the dashboard. Use this report to help identify the letters returned.</p>
```

Comment:
I’m suggesting this change because:

* the first sentence could be more active
* the second sentence kinda repeats the first – or provides detail that is implicit in the first sentence

```suggestion
  <p class="govuk-body">You can see a report of returned letters on the dashboard.</p>
```